spring-cloud-gcp-data-spanner/src/main/java/com/google/cloud/spring/data/spanner/core/SpannerOperations.java

```diff
@@ -49,6 +50,15 @@ public interface SpannerOperations {
    * @return the lower-bound of number of rows affected.
    */
   long executePartitionedDmlStatement(Statement statement);
+  /**
+   * Execute a DML statement in partitioned mode. This is not available inside of transactions.
+   *
+   * @param statement the DML statement to execute.
+   * @param options marks options applicable to update operation.
+   * @return the lower-bound of number of rows affected.
+   */
+
+  long executePartitionedDmlStatement(Statement statement, UpdateOption... options);
```

Comment:
Spacing:
```suggestion

  /**
   * Execute a DML statement in partitioned mode. This is not available inside of transactions.
   *
   * @param statement the DML statement to execute.
   * @param options marks options applicable to update operation.
   * @return the lower-bound of number of rows affected.
   */
  long executePartitionedDmlStatement(Statement statement, UpdateOption... options);
```

Reply:
@meltsufin Sure, somehow `fmt-maven-plugin` was acting strange! I have fixed the spacing